downstream/modules/hub/proc-add-user-to-group.adoc

```diff
@@ -15,8 +14,8 @@ Complete the following steps to add users to an existing group.
 
 . Log in to your local {HubName}.
 . From the navigation panel, select menu:User Access[Groups].
-. Click on a Group name.
-. Navigate to the menu:Users[] tab, then click btn:[Add].
+. Click on a *Group* name.
+. Navigate to the *menu:*Users* tab and click btn:[Add].
```

Comment:
Is there an extra asterisk (*) here i.e. 3 when there should be 2?

Reply:
Fixed.